From 1090a8826a8435196554ac16f3b8a01cbb03f4be Mon Sep 17 00:00:00 2001
From: Dan Fuhry <dfuhry@datto.com>
Date: Thu, 25 Aug 2016 17:13:52 -0400
Subject: [PATCH] livenet: support nfs:// urls in livenet-generator

This is a further improvement on PR #105 which re-adds support for nfs:// urls to root=live:nfs://... Symptoms prior to applying this patch are that sysroot.mount times out when following the nfs:// syntax for the path to the live image. An additional case is added to livenet-generator to support the nfs protocol.
---
 modules.d/90livenet/livenet-generator.sh | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/modules.d/90livenet/livenet-generator.sh b/modules.d/90livenet/livenet-generator.sh
index 377d53b..969ef4f 100755
--- a/modules.d/90livenet/livenet-generator.sh
+++ b/modules.d/90livenet/livenet-generator.sh
@@ -16,6 +16,9 @@ fi
 [ "${liveroot%%:*}" = "live" ] || exit 0
 
 case "$liveroot" in
+    live:nfs://*|nfs://*) \
+        root="${root#live:}"
+        rootok=1 ;;
     live:http://*|http://*) \
         root="${root#live:}"
         rootok=1 ;;
